src/analyzer/protocol/modbus/modbus-protocol.pac

```diff
@@ -112,7 +112,7 @@ type ModbusTCP_Request(header: ModbusTCP_TransportHeader) = case header.fc of {
 #
 type ModbusTCP_Response(header: ModbusTCP_TransportHeader) = case header.fc & 0x80 of {
     0       -> normal_response      : ModbusTCP_NormalResponse(header);
-    default -> exception_response   : Exception(header);
+    default -> exception_response   : ExcResponse(header);
```

Comment:
Nit. Given the normal response is called `ModbusTCP_NormalResponse`, `ModbusTCP_ExceptionResponse` might be a candidate that pleases clang-tidy and doesn't look artificially shortened?

Reply:
clang-tidy is very picky about naming things with `exception` in their name:

```
[6/41] Building CXX object src/analyzer/protocol/modbus/CMakeFiles/plugin-Zeek-Modbus.dir/modbus_pac.cc.o
/Users/tim/Desktop/projects/zeek-master/build/src/analyzer/protocol/modbus/modbus_pac.cc:585:43: warning: suspicious exception object created but not thrown; did you mean 'throw ModbusTCP_ExceptionResponse'? [bugprone-throw-keyword-missing]
```

Reply:
But it does seem happy with `ModbusTCP_ExceptResponse` :shrug: